### PR TITLE
support sending logs to a graylog server

### PR DESCRIPTION
**Pull request by ayan-usgs**
Before making a pull request
----------------------------

- [x] Make sure all tests run
- [x] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [x] If appropriate, put the link to the PR in the JIRA ticket
- [x] Assign someone to review unless the change is trivial

### CODE REVIEW

File: server/config.py.sample [hunk 1]
```diff
@@ -26,6 +26,9 @@ ROBOTS_WELCOME = False
 # If logging is turned on or not
 LOGGING_ON = False
 
+# Graylog configuration
+GRAYLOG_CONFIG = {'host': '127.0.0.1', 'port': 12201}
```
Comment (mbucknell) [inline server/config.py.sample]: I don't think this needs to go into the sample. The sample is used by the make files to initialize instance/config.py if one doesn't exist, the logging would likely through errors.

File: server/pubs_ui/__init__.py [hunk 2]
```diff
@@ -9,16 +9,39 @@
...
+    If no arguments are provided, the logs will be streamed to stdout using StreamHandler.
+
+    :param dict graylog_config: optional dictionary with `host` and `port` keys, which specify
+        the Graylog server's host and the port being listened to. For example, `{'host': '127.0.0.1', 'port': 12201}`.
+    :param log_level: the log level to use
+    :return: a handler
+    :rtype: logging.Handler
+
+    """
+    if graylog_config is not None:
```
Comment (mbucknell) [inline server/pubs_ui/__init__.py]: Can we do have both Graylog and streaming? I'd like to continue to stream logs to standard out even if Graylog is being used. I believe the services are set up this way.

Comment (ayan-usgs) [reply server/pubs_ui/__init__.py]: Yep.

File: server/config.py.sample (hunk 1 above)
Comment (ayan-usgs) [reply server/config.py.sample]: Removed.

**Pull request merged**